### PR TITLE
[KeyVault Certs] Fix cspell errors

### PR DESCRIPTION
**Pull request by YalinLi0312**
For issue: https://github.com/Azure/azure-sdk-for-python/issues/22661

### CODE REVIEW

**Review by mccoyp — APPROVED** [review]
This is a huge improvement, thank you! Again, might want to wait for @danieljurek's approval before merging.

**Review by danieljurek — APPROVED** [review]
🚢 ❗

**Pull request merged**
